src/SecurityManager.ts

```diff
@@ -278,14 +277,7 @@ async function doAccessSecretStorage(func: () => Promise<void>, forceReset: bool
             await crypto.bootstrapSecretStorage({});
 
             const keyId = Object.keys(secretStorageKeys)[0];
-            if (keyId && SettingsStore.getValue("feature_dehydration")) {
-                let dehydrationKeyInfo = {};
-                if (secretStorageKeyInfo[keyId] && secretStorageKeyInfo[keyId].passphrase) {
-                    dehydrationKeyInfo = { passphrase: secretStorageKeyInfo[keyId].passphrase };
-                }
-                logger.log("accessSecretStorage: Setting dehydration key");
-                await cli.setDehydrationKey(secretStorageKeys[keyId], dehydrationKeyInfo, "Backup device");
-            } else if (!keyId) {
+            if (!keyId) {
```

Comment:
This whole `if` statement is related to dehydration, so can be removed completely -- the remaining branches are just logging, explaining why it isn't setting a decryption key

Reply:
which means `keyId` can also be removed, since it isn't being used for anything else